covalent/_workflow/transport.py

```diff
@@ -82,6 +82,9 @@ def encode_metadata(metadata: dict) -> dict:
         else:
             encoded_metadata["triggers"] = metadata["triggers"]
 
+    # qelectron_data_exists
+    encode_metadata["qelectron_data_exists"] = None
```

Comment:
1) Do you mean `encoded_metadata`?
2) Should the `qelectron_data_exists`  property be referenced in the electron decorator?

Reply:
1. Yep, had already updated that
2. Not really, it's similar to `executor_data`, since we need to have it as a metadata so I thought it should have a default value, which I'm leaning towards being `False` instead of `None` since in the migrations script it is a non-nullable column.